app/services/request_reference.rb

```diff
@@ -31,4 +45,19 @@ def auto_approve_reference(reference)
   def email_address_is_a_bot?(reference)
     /^refbot(\d+)@example.com/ =~ reference.email_address
   end
+
+  def reference_complete_information
+    errors.add(:reference, :incomplete_references) if incomplete_reference?
+  end
+
+  def incomplete_reference?
+    !CandidateInterface::Reference::SubmitRefereeForm.new(
+      submit: 'yes',
+      reference_id: @reference.id,
+    ).valid?
+  end
+
```

Comment:
could we ask the positive question here? `errors.add(...) unless complete_reference?`

Reply:
Sure! I'll do it

Reply:
Done! ♻️